Jenkinsfile

```
@@ -56,33 +66,92 @@ def get_daos_packages(String distro) {
     return pkgs + "-" + daos_packages_version(distro)
 }
 
-def component_repos() {
+def pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos')
 }
 
-String el7_component_repos() {
+String el7_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-el7')
 }
 
-String leap15_component_repos() {
+String leap15_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-leap15')
 }
```

Per below, I cannot think of a reason why this function doesn't return both the all-distro and per-distro `PR-repos` values:

```suggestion
def pr_repos() {
    String repos = ""
    if (distro == 'centos7') {
￼        repos = cachedCommitPragma(pragma: 'PR-repos-el7')
￼    } else if (distro == 'leap15') {
￼        repos = cachedCommitPragma(pragma: 'PR-repos-leap15')
￼    } else {
         error 'pr_repos not implemented for ' + distro
    }

    return cachedCommitPragma(pragma: 'PR-repos') + ' ' + repos
}
```